apps/explorer/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -43,11 +50,14 @@ function OwnedObject({ id, byAddress }: { id: string; byAddress: boolean }) {
 
         req.then((objects) => {
             let ids: string[];
-            if (objects instanceof Array) {
-                ids = objects.map(({ objectId }) => objectId);
+            if (isPaginatedObjectsResponse(objects)) {
+                ids = objects.data.map(( {details} ) => {
+                    const sui_object_data = details as SuiObjectData;
+                    return sui_object_data.objectId;
+                });
```

Comment:
```suggestion
                ids = objects.data.map(resp => getObjectId(resp));
```

you also need to import `getObjectId` from '@mysten/sui.js'. We prefer to use getter method rather than accessing the field directly because then we can have schema change without updating the codebase here